src/adb/forward.js

```diff
@@ -0,0 +1,19 @@
+let debug = console.log.bind(console, '[adb/forward]');
+let exec = require('mz/child_process').exec;
```

Comment:
I think the `mz` dependency is ok for now, but once we need a consolidated interface for `exec` for common options, it will most likely only be used in a single place.